frontend/src/lib/components/TaxonomicFilter/InlineHogQLEditor.tsx

```diff
@@ -9,18 +10,47 @@ export interface InlineHogQLEditorProps {
     metadataSource?: AnyDataNode
 }
 
+const SHORTCUTS = [
```

Comment:
Guessing can expose things to use latest event/person property
```
select 
   event, 
   pdi.person.properties.email, -- current property value, fetched via a join
   poe.properties.email, -- ingestion time property stored on the event
   person.properties.email -- whatever is the default setting for your team, one of the two above
from events limit 10
```

Reply:
Skipping these for now as we'll implement this in the event/person properties picker